packages/telnyx_webrtc/lib/peer/peer.dart

```diff
@@ -348,14 +353,14 @@ class Peer {
     onDataChannel?.call(session, channel);
   }
 
-  Future<void> _createDataChannel(Session session,
+  /*Future<void> _createDataChannel(Session session,
```

Comment:
Hey @Oliver-Zimmerman, are these functions (`_createDataChannel` and `_closeSessionByPeerId`) commented intentionally?

Reply:
@KalpeshTalkar Yeah, we may need them later for later features, however, static code analysis doesn't like uncalled methods and brings our pub score down. I am leaving them commented out instead for now!

Reply:
Got it. Thanks for the explanation :)